cl_tf2/src/cl-tf2/stamped-data.lisp

```diff
@@ -71,6 +71,7 @@
    - define a class 'number-stamped' with slots 'num-value' and 'header'
    - define a constructor-function 'make-number-stamped'
    - define a function 'copy-number-stamped'
+   - define a function 'copy-ext-number-stamped'
```

Comment:
Tried it today. Apparently,

```lisp
(defgeneric method-name (param-1 param-2 &key &allow-other-keys)
  (:method (param-1 param-2 &key key-1 key-2)
    ...)
  (:method (param-1 param-2 &key key-3)
    ...)
```
doesn't work, as it will default to the first method defined. Thus, these methods are not differentiated by the keys specified when calling `method-name`.

I'll therefore for now stick to `copy-ext-...`.